Drop-In/src/main/java/com/braintreepayments/api/PaymentMethodInspector.java

```diff
@@ -0,0 +1,112 @@
+package com.braintreepayments.api;
+
+import com.braintreepayments.cardform.utils.CardType;
+
+class PaymentMethodInspector {
+
+    private static final String PAYMENT_METHOD_AMEX = "American Express";
+    private static final String PAYMENT_METHOD_DINERS_CLUB = "Diners";
+    private static final String PAYMENT_METHOD_DISCOVER = "Discover";
+    private static final String PAYMENT_METHOD_GOOGLE_PAY = "Google Pay";
+    private static final String PAYMENT_METHOD_HIPER = "Hiper";
+    private static final String PAYMENT_METHOD_HIPERCARD = "Hipercard";
+    private static final String PAYMENT_METHOD_JCB = "JCB";
+    private static final String PAYMENT_METHOD_MAESTRO = "Maestro";
+    private static final String PAYMENT_METHOD_MASTERCARD = "MasterCard";
+    private static final String PAYMENT_METHOD_PAYPAL = "PayPal";
+    private static final String PAYMENT_METHOD_UNION_PAY = "UnionPay";
+    private static final String PAYMENT_METHOD_VENMO = "Venmo";
+    private static final String PAYMENT_METHOD_VISA = "Visa";
+
+    String getPaymentMethodDescription(PaymentMethodNonce paymentMethodNonce) {
```

Comment:
We should move `getPaymentMethodDescription()` and `getPaymentMethod()` methods into `PaymentMethodNonce` for the next major version.

Reply:
Is this approach then to avoid further delay in GA by not relying on an upstream change/merge?  Just trying to weigh the pros/cons on this one.

Reply:
Yeah pretty much. It is internal though, so if we decided to add those methods in core we can gradually remove this class without breaking changes.